### DISCUSSION TITLE
Discussions are not showed up when moved from removed category

### DISCUSSION THREAD

**Discussion by AlenaSviridenko** [community/community#3030] [Discussions]
Hey,
I faced with weird behavior:
I created a new category and removed the old one, and moved discussion from the old category to the new one.
Discussion has an emoji of new category but when you select this new category in the list on the left - discussion is not appeared there. 
Unassign/re-assign category on a discussion page helped.

**Comment by jdrch** [comment]
Same problem and solution here.

**Comment by vmatare** [comment]
Same problem here, but re-assigning isn't helping:

https://github.com/vmatare/thinkfan/discussions/121

No matter where I assign it, it never shows up in any category.
